app/build.gradle

```diff
@@ -54,6 +80,37 @@ sourceSets {
     main.java.srcDirs += "${protobuf.generatedFilesBaseDir}/main/javalite"
 }
 
+// Map for the version code that gives each ABI a value.
+ext.abiCodes = ['armeabi-v7a': 1, 'arm64-v8a': 2]
+
+// For each APK output variant, override versionCode with a combination of
+// ext.abiCodes * 1000 + variant.versionCode. In this example, variant.versionCode
```

Comment:
Using 1000 as high value would leave us with a release every week for about 20 years. Looks legit to me.

Reply:
yes and in 20 years we can still increase it to 10.000